stonesoup/dataassociator/neighbour.py

```diff
@@ -101,3 +101,41 @@ def associate(self, tracks, detections, time):
         associations = max(joint_hypotheses)
 
         return associations
+
+
+class GaussianMixtureAssociator(DataAssociator):
```

Comment:
Don't believe this is required, as could have tracker just use a Hypothesiser.

Reply:
Ok, removed.